scipy/spatial/transform/rotation.py

```diff
@@ -243,9 +250,13 @@ def from_quaternion(cls, quat, normalized=False):
             If this flag is `True`, then it is assumed that the input
             quaternions all have unit norm and are not normalized again.
             Default is False.
+        copy : boolean, optional
```

Comment:
Why do you believe this flag is necessary here? I thought it might be set to `True` automatically when the object is created through this function.

Reply:
I don't think it's necessary _per se_, but since it's `True` by default, I think it was ok to allow it here. Should we remove it completely?

Reply:
As there is no immediate value in it I suggest to remove it at least for now for simplicity.